### PR TITLE
Lowered the intensity of Blur on hover

### PR DESCRIPTION
**Pull request by thecodereule**
**What does this PR do?**
Lowers the amount of blur effect value present when hovering over an image you are searching for

**Related Issue(s)**
related to issue #92 

**Changes**
modified tailwind.config.tx by adding a backdropBlur configuration line

**Screenshots or GIFs (if applicable)**
image

**Checklist**
- [ x] I have read and followed the [contributing guidelines](../CONTRIBUTING.md).
- [ x] My code follows the project's coding style and conventions.
- [ x] I have added unit tests (if applicable) to ensure the code works as expected.
- [ x] I have updated the project documentation (if necessary).

**Additional Comments**
Add any additional comments, context, or information about this PR.

**By submitting this PR, I confirm that:**
- [ x] I have reviewed my code and believe it is ready for merging.
- [ x] I understand that this PR may be subject to review and changes.
- [ x] I agree to abide by the code of conduct and contributing guidelines of this project.

### CODE REVIEW

**Comment by Vijaykv5** [comment]
Don't see the real change
@thecodereule

**Comment by thecodereule** [comment]
> Don't see the real change @thecodereule

Do you mean you don't see the code that was changed?
image

File: tailwind.config.ts [hunk 1]
```diff
@@ -3,6 +3,9 @@ module.exports = {
   content: ['./src/**/*.{html,js,ts,jsx,tsx}'],
   theme: {
     extend: {
+      backdropBlur: {
```
Comment (thecodereule) [inline tailwind.config.ts:6]: changed the backdrop blur value from 4px to 1.5px

Comment (Vijaykv5) [reply tailwind.config.ts:6]: Okay

**Pull request merged**